src/globus_sdk/globus_app/app.py

```diff
@@ -178,6 +177,26 @@ def _initialize_login_client(
         Initializes and returns an AuthLoginClient to be used in authorization requests.
         """
 
+    def _initialize_validating_token_storage(
+        self,
```

Comment:
It looks like this can be a static method, but I don't know how that will affect subclasses. Is adding `@staticmethod` worth considering here?

Reply:
I don't think so. This structure is inline with the rest of the class & gives us the ability to reference the instance being instantiated if we need it in the future.

What benefit would we see from making this method static?

Reply:
Nothing technical that I'm aware of, I'm just attuned to it because PyCharm complains by default.